runtime/service/src/main/java/org/apache/polaris/service/quarkus/events/QuarkusPolarisEventListenerConfiguration.java

```diff
@@ -20,13 +20,29 @@
 
 import io.quarkus.runtime.annotations.StaticInitSafe;
 import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+import java.util.Optional;
+import org.apache.polaris.service.events.EventListenerConfiguration;
+import org.apache.polaris.service.events.PolarisEventListener;
 
 @StaticInitSafe
 @ConfigMapping(prefix = "polaris.event-listener")
-public interface QuarkusPolarisEventListenerConfiguration {
+public interface QuarkusPolarisEventListenerConfiguration extends EventListenerConfiguration {
   /**
-   * The type of the event listener to use. Must be a registered {@link
-   * org.apache.polaris.service.events.PolarisEventListener} identifier.
+   * The type of the event listener to use. Must be a registered {@link PolarisEventListener}
+   * identifier.
    */
   String type();
+
+  @WithName("aws-cloudwatch.log-group")
+  @Override
+  Optional<String> awsCloudwatchlogGroup();
```

Comment:
My same comment about extensibility applies; I would expect a new `...Configuration` type per `PolarisEventListener` implementation

Reply:
This is probably my biggest remaining concern after the recent changes... I just imagine that with the 3+ event listeners you're proposing these config types will get quite messy.